src/pkg/quota/models/quota.go

```diff
@@ -86,3 +87,39 @@ func (q *Quota) SetUsed(used types.ResourceList) *Quota {
 
 	return q
 }
+
+// GetWarningResources returns resource names which exceeded the warning percent
+func (q *Quota) GetWarningResources(warningPercent int) ([]types.ResourceName, error) {
```

Comment:
should only send out the current project quota message, excludes others.

Reply:
The `Quota` belongs the project, `GetWarningResources` is ok here.